plat/kvm/include/kvm-x86/multiboot.h

```diff
@@ -20,9 +20,10 @@
  */
 /* multiboot.h - Multiboot header file.  */
 
-#ifndef MULTIBOOT_HEADER
-#define MULTIBOOT_HEADER 1
```

Comment:
Not related to this PR but, maybe in the future, would it make sense to move the multiboot headers to a platform common directory? I do not think it would make sense to have it in a KVM only directory since, for example, Xen also supports it. Perhaps even architecture indepedent (especially for Multiboot2) as at one point, iirc, there were some Xen patches to make Multiboot more generic and accomodate for ARM.

Reply:
Yes, absolutely. We are currently in a platform re-arch. I already laid out the files and most of the code so that we can easily move this into a dedicated `boot-multiboot` library. This would exist together with, for example, `boot-efi` etc. For the PR, I'll keep the header where it is and we move it out when we create the boot library.

Updates regarding multibootv2 we would then either incorporate in the same library and headers or have another one specific for v2.